HW_1/2.py

```diff
@@ -0,0 +1,7 @@
+n = int(input('Введите трёхзначное число: '))
+sum = 0
+while n > 0:
+    x = n % 10
+    sum += x
```

Comment:
Можно совместить эти 2 строки. От этого читаемость не ухудшится.
```suggestion
    sum += n % 10
```